packages/docs/src/locales/en/en.json

```diff
@@ -332,6 +332,7 @@
         "overlayOpacity": "Set the overlay's opacity",
         "anchorClass": "Set class name to the `anchor` slot container",
         "zIndex": "Set the modal's `z-index`",
+        "fixBody": "Set `body` position to `fixed` to prevent document scroll while modal is open.",
```

Comment:
Imo, poor naming. I understand that this property adds `position: fixed`, but it looks like we fixing something in the `document.body`. I think there will fit `preventScroll` or something similar.

Reply:
preventPageScroll sounds good.